Source/Interface/Tips.js

```diff
@@ -170,15 +172,22 @@ this.Tips = new Class({
 		clearTimeout(this.timer);
 		this.timer = (function(){
 			this.container.empty();
-
+			var showTip = !this.options.hideEmpty;
 			['title', 'text'].each(function(value){
 				var content = element.retrieve('tip:' + value);
 				var div = this['_' + value + 'Element'] = new Element('div', {
 						'class': 'tip-' + value
 					}).inject(this.container);
-				if (content) this.fill(div, content);
+				if (content){
+					this.fill(div, content);
+					showTip = true;
+				}
 			}, this);
-			this.show(element);
+			if (showTip){
+				this.show(element);
+			} else {
+				this.hide(element);
+			}
```

Comment:
minor style thing; MooTools style is to put these on one line:
```javascript
if (true) something();
else somethingElse();
```
only if either of the conditional statements are > 1 line do we add braces (in which case we add braces to both clauses, even if one is one line).

At least, that used to be the convention. Again, not a blocker on shipping it.